handlers/conversation_handlers.go

```diff
@@ -50,6 +50,39 @@ func (env *Env) PostConversationHandler(w http.ResponseWriter, r *http.Request)
 	json.NewEncoder(w).Encode(reqConversation)
 }
 
+// GetUsersConversationsHandler returns all of a user's conversations
+func (env *Env) GetUsersConversationsHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
I think this should be called `GetConversationsHandler` to make it more consistent with the other method names.